### PR TITLE
cleanup code for stackdriver exporter registration.

### PR DESCRIPTION
**Pull request by rghetia**

### CODE REVIEW

File: src/adservice/src/main/java/hipstershop/AdService.java [hunk 1]
```diff
@@ -226,6 +221,12 @@ public static void main(String[] args) throws IOException, InterruptedException
 
     initializeAds();
 
+    // Registers all RPC views.
+    RpcViews.registerAllViews();
+
+    // Registers logging trace exporter.
+    LoggingTraceExporter.register();
```
Comment (sebright) [inline src/adservice/src/main/java/hipstershop/AdService.java]: Why did you move this section of code?  I thought it was better to register the views earlier to avoid losing data.

Comment (rghetia) [reply src/adservice/src/main/java/hipstershop/AdService.java]: I have two exporters, Prometheus and Stackdriver, initializing views  within initStackdriver() makes it look like registeration of views is part of Stackdriver. In fact it is common to both Stackdriver and Prometheus. So I moved it common place in main().

Comment (sebright) [reply src/adservice/src/main/java/hipstershop/AdService.java]: Sorry, I misread the code and thought this section was moved farther down in initStackdriver.  This change is how I expected initialization to work.
